docs/maps/import-geospatial-data.asciidoc

```diff
@@ -6,6 +6,30 @@ To import geospatical data into the Elastic Stack, the data must be indexed as {
 Geospatial data comes in many formats.
 Choose an import tool based on the format of your geospatial data.
 
+[discrete]
+[[import-geospatial-privileges]]
+=== Security privileges
+
+The {stack-security-features} provide roles and privileges that control which users can upload files.
+You can manage your roles, privileges, and
+spaces in the **{stack-manage-app}** app in {kib}. For more information, see
+{ref}/security-privileges.html[Security privileges],
+<<kibana-privileges, {kib} privileges]>>, and <<xpack-kibana-role-management, {kib} role management>>.
+
+To upload GeoJson files in {kib} with *Maps*, you must have:
+* [ ] `all` {kib}
```

Comment:
This list isn't formatting correctly.  I think it's the use of the brackets.  What if you replaced [ ] with the word "The"?